core/storage/storage_models_test.py

```diff
@@ -49,9 +49,11 @@ def _get_model_classes(self):
                     if 'Model' in all_base_classes:
                         yield clazz
 
-    # List of model classes that can't have Wipeout related class methods
+    # List of model classes that don't have Wipeout related class methods
     # defined because they're not used directly but only as a base classes for
-    # the other models.
+    # the other models. BaseCommitLogEntryModel is not included in this list
```

Comment:
I think this reads a bit more cleanly if you put two line breaks after `models.`

Reply:
Done.